### PR TITLE
GitHub marketplace requires naming changes

### PR DESCRIPTION
**Pull request by kuritka**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-name: Setup K3d
-description: Easy to use GitHub action allows you to test single/multi-cluster apps with additional private registry support.
+name: AbsaOSS/k3d-action
+description: A GitHub Action to run lightweight ephemeral Kubernetes clusters during workflow..
```
Comment (donovanmuller) [inline action.yaml]:
```suggestion
description: A GitHub Action to run lightweight ephemeral Kubernetes clusters during workflow
```

**Review by k0da — APPROVED** [review]

**Review by donovanmuller — COMMENTED** [review]
@kuritka What makes this action different to the [existing actions](https://github.com/marketplace?query=k3d)? What I mean is, you should note the unique selling points or differentiators of this action vs. the others.

End users might appreciate reading what makes this action unique and why they should use this action vs. others.

**Review by somaritane — APPROVED** [review]

**Comment by kuritka** [comment]
@donovanmuller, sure, where we should mention that ?


Here are some arguments:

- The fundamental problem for the rest of k3d actions is (except some of them are [not working at all (setup nolar/k3d/k3s - the most of the code is some parsing version with jq, which actually fails)](https://github.com/nolar/setup-k3d-k3s)), you can't pass freely any argument from [k3d command tree](https://k3d.io/usage/commands/) or any [k3s argument](https://rancher.com/docs/k3s/latest/en/installation/install-options/server-config/#k3s-server-cli-help).  For instance  [this one (rinx/setup-k3d)](https://github.com/rinx/setup-k3d/blob/master/entrypoint.sh) hard-codes port on hard-coded load-balancer. For the real scenarios, we need highly customizable action.

- Only our action provides embedded docker registry (kind actions don't provide).
- Only our action creates clusters out of default docker bridge network  (kind actions don't provide), it does not limit us in multi-cluster scenarios. 
- k3d-action beats kind actions compared to memory requirements and performance. Therefore, it is suitable for multi-node, multi-cluster scenarios. 
- All scenarios are documented and tested by action itself

**Comment by donovanmuller** [comment]
@kuritka based on:

> The fundamental problem for the rest of k3d actions is (except some of them are not working at all (setup nolar/k3d/k3s - the most of the code is some parsing version with jq, which actually fails)), you can't pass freely any argument from k3d command tree or any k3s argument. For instance this one (rinx/setup-k3d) hard-codes port on hard-coded load-balancer. For the real scenarios, we need highly customizable action.

That is your differentiator. More customisable. If you could work a minimal version of the above into the description and then `README` that should be fine.

**Comment by kuritka** [comment]
@donovanmuller What about this? 

_Fundamental advantage of this GitHub action is a full customization of embedded k3s clusters. In addition, it provides a private image registry and multi-cluster support._

**Comment by donovanmuller** [comment]
💯

**Pull request merged**